bigframes/session/__init__.py

```diff
@@ -977,7 +976,7 @@ def _read_pandas(
         job_config.clustering_fields = cluster_cols
         job_config.labels = {"bigframes-api": api_name}
 
-        load_table_destination = self._create_session_table()
+        load_table_destination = bigframes_io.random_table(self._anonymous_dataset)
         load_job = self.bqclient.load_table_from_dataframe(
             pandas_dataframe_copy,
             load_table_destination,
```

Comment:
Need to update the `table_expression` below. We don't have to do `ibis_client.sql(...)` anymore, we can use a real table expression.